time.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+function durationBetweenDates (initalDate = "14 Jan 1992", endDate = "11 March 2023", timeUnit = "days") {
+    let duration;
+    const timeUnitsTable = {
+        seconds: 1000,
+        minutes: 60000,
+        hours: 3600000,
+        days: 86400000
+    }
+    const milisecInitalDate = new Date(initalDate);
+    const milisecEndDate = new Date(endDate);
+    let milisecDif = milisecEndDate - milisecInitalDate;
+    if (milisecDif < 0) {
+        milisecDif *= -1;
+    };
+    switch (true) {

```

Comment:
`switch (true)` is a very very bad construction. Try to avoid it.

Reply:
Replaced